packages/cli/.gitignore

```diff
@@ -1 +1,112 @@
+# Logs
```

Comment:
I'd remove this generic gitignore and replace it with a minimal version that only ignores what actually needs to be ignored.

Reply:
Done